### PR TITLE
Homework 08.11.2024 first commit

### PR DESCRIPTION
**Pull request by HKV203**

### CODE REVIEW

File: src/main/java/com/andersen/bookstore/controller/Controller.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    private void saveLastState(Bookstore bookstore, String filepath) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
+        try {
+            File file = new File(filepath);
+            objectMapper.writeValue(file, bookstore.getOrders());
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+        }
+    }
```
Comment (Alexmansar) [inline src/main/java/com/andersen/bookstore/controller/Controller.java]: I would move it to anotherclass

File: src/main/java/com/andersen/bookstore/view/Menu.java [hunk 2]
```diff
@@ -17,32 +17,37 @@ public Menu(Bookstore bookstore) {
...
+                break;
+            case 2:
+                showOrdersList();
+                break;
+            case 3:
+                showAvailabilityMenu();
+                break;
+            default:
+                break;
+
```
Comment (Alexmansar) [inline src/main/java/com/andersen/bookstore/view/Menu.java]: use new style please

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@Mopgonpblr 
Hi)) in general good job)) I have some comments)so please, check them)) And add please new tests for new logic)

File: src/main/java/com/andersen/bookstore/view/Menu.java (hunk 2 above)
Comment (HKV203) [reply src/main/java/com/andersen/bookstore/view/Menu.java]: Does it mean to replace the switch-case construction? If so then with what?

Comment (Alexmansar) [reply src/main/java/com/andersen/bookstore/view/Menu.java]: https://www.baeldung.com/java-switch#1-the-new-switch-expression

**Review by Alexmansar — APPROVED** [review]
@Mopgonpblr Please, merge it)) good job)

**Pull request merged**